modules/platform/nuxeo-platform-video/src/main/java/org/nuxeo/ecm/platform/video/VideoHelper.java

```diff
@@ -255,4 +266,26 @@ public static VideoInfo getVideoInfo(Blob video) {
         }
     }
 
+    /**
+     * Commits and starts a new transaction with a custom timeout.
+     *
+     * @since 11.5
+     */
+    public static void newTransaction() {
+        if (TransactionHelper.isTransactionActiveOrMarkedRollback()) {
+            TransactionHelper.commitOrRollbackTransaction();
+        }
+        if (log.isDebugEnabled()) {
+            log.debug("Commit and start transaction with timeout " + getTransactionTimeout() + "s");
```

Comment:
Worth extracting `getTransactionTimeout()` to a var?

Reply:
not necessary for a debug trace